modules/core/arrow-core-data/src/main/kotlin/arrow/typeclasses/Bitraverse.kt

```diff
@@ -0,0 +1,50 @@
+package arrow.typeclasses
+
+import arrow.Kind
+import arrow.Kind2
+import arrow.core.Id
+import arrow.core.identity
+import arrow.core.value
+import arrow.documented
+import arrow.typeclasses.internal.IdBimonad
+
+/**
+ * ank_macro_hierarchy(arrow.typeclasses.Bitraverse)
+ *
+ * The type class `Bitraverse` defines the behaviour of two separetes `Traverse` over a data type.
+ *
+ * Every instance of `Bitraverse<F>` must contains the next functions:
+ *
+ * ## Bitraverse
+ *
+ * `Bitraverse` perfoms a`Traverse` over both side of the Data type which is `Bifoldable`.
+ *
+ * ```kotlin
```

Comment:
these should be ank playgrounds similar to how it's done in Functor or Id https://github.com/arrow-kt/arrow/blob/38703b5a7b9477892be5093109d5421b85ac2dac/modules/core/arrow-core-data/src/main/kotlin/arrow/core/Id.kt#L13